communication/slack_channels.md

```diff
@@ -0,0 +1,78 @@
+# Slack Channels
+
+Please find a list here of Slack channels at source{d}.
+
+Slack channels are encouraged to be open for everyone to join. Private channels for professional reasons are highly discouraged (have fun with private personal slack channels). We encourage people to have discussions in open/public channels rather than direct messages when the information is relevant to multiple people for a limited amount of time. If a piece of information should be preserved, it should always go to the guide or company repo. 
```

Comment:
This "everyone" is ambiguous. Does it mean all source{d} employees, or all human beings currently alive?